### ISSUE TITLE
Why would this package require gcc, g++ and python?

### ISSUE THREAD

**Issue by jasworks** [homebridge/homebridge-apt-pkg#8]
### 

Hi, just a question, I noticed that this debian package requires gcc, g++ and python3.

Just wondering why? Shouldn’t homebridge be able to run with pure Node.js?

### Logs

```markdown
N/A
```

### Configuration

```json
Debian
```

### Environment

- OS: LInux, Debian-based self built docker image
- Software: Debian, Docker
- Node:
- npm:

### Process Supervisor

hb-service

**Comment by NorthernMan54** [comment]
@jasworks You are correct, Homebridge itself uses native nodeJS, but some plugins do require the usage gcc, g++ and python3.  Hence we have included them.
